tcms/core/tests/test_middleware.py

```diff
@@ -0,0 +1,19 @@
+import os
+import unittest
+from django.test import TransactionTestCase
+
+
+@unittest.skipUnless(
+    os.getenv('TEST_CHECK_UNAPPLIED_MIGRATIONS_MIDDLEWARE'),
+    'CheckUnappliedMigrationsMiddleware testing not enabled')
+class TestCheckUnappliedMigrationsMiddleware(TransactionTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.unapplied_migration_message = "unapplied migration(s). See "\
+            "<a href=\"https://kiwitcms.readthedocs.io/en/latest/"\
+            "installing_docker.html#initial-configuration-of-running-"\
+            "container\">documentation</a>"
+
+    def test_unapplied_migrations(self):
+        response = self.client.get('/', follow=True)
+        self.assertContains(response, self.unapplied_migration_message)
```

Comment:
The new test is failing in CI, make sure that is passes. You can print the response conents here to figure out what page it is. 

I'm not sure how TransactionTestCase will work in this situation, almost everywhere else we use test.TestCase which inherits from TransactionTestCase. You may need to change as well, IDK.

Reply:
Not seeing the message in the response, only seeing the Base URL not configured message